app/tests/spec/lib/experiments/grouping-rules/token-code.js

```diff
@@ -31,10 +31,6 @@ define(function (require, exports, module) {
       });
 
```

Comment:
Also need to bring back this test https://github.com/mozilla/fxa-content-server/pull/5913/files#diff-b41a82ac6da6d2b1fad23eefe3f6acd2  right?